recipe/meta.yaml

```diff
@@ -20,19 +20,21 @@ source:
     - osx.patch  # [osx]
 
 build:
-  number: 0
+  number: 1
   detect_binary_files_with_prefix: true
   skip: true  # [win]
 
 requirements:
   build:
     - gcc  # [osx]
+    - libgcc  # [osx]
```

Comment:
Are you sure? `libgcc` wasn't pulled in during the first OSX build of this package, and the resulting library has a busted dependency on libgcc_s: [see this line of output from the OSX build](https://travis-ci.org/conda-forge/pgplot-feedstock/builds/196165092#L1046), and then look at what packages it installed at the beginning of the build.